File: help/eng_gb/exercicios_da_ficha_3/Ficha03_019/Ficha03_019.php
<?php
// Programmer           Ant0ni0 M@ns0 - Ant0ni0 M@ns0

//Main Function
function main() {
    echo  "Angle (rad)     : " ;
    $ang = floatval(isset($_POST["ang"])? $_POST["ang"] : isset($_GET["ang"]) ? $_GET["ang"] : 0);
    
    echo $ang . "<br/>";// print input 
    echo  "Sin         = " . ( sin( $ang )  ) . "\n" ;
    echo  "Cosine         = " . ( cos( $ang )  ) . "\n" ;
    echo  "Tangent     = " . ( tan( $ang )  ) ;
}

main();

//PHP 5.3 +  - Automatic translated by Algorithmi 22.05
//(c) Luís Lopes 	Tue, 2022 Sep 13 21:58

?>

File: help/eng_gb/exercicios_da_ficha_3/Ficha03_019/Ficha03_019.java
// Programmer           Ant0ni0 M@ns0 - Ant0ni0 M@ns0

import java.util.Scanner;

public class Ficha03_019 {

    static Scanner keyb = new Scanner(System.in);//keyboard

    //Main Function
    public static void main( String[] args) {
        System.out.print("Angle (rad) \t: " );
        double ang = Double.valueOf(keyb.nextLine());
        System.out.print("Sin \t\t= " + Math.sin( ang ) + "\n" );
        System.out.print("Cosine \t\t= " + Math.cos( ang ) + "\n" );
        System.out.print("Tangent \t= " + Math.tan( ang ) );
    }

}
//Java  - Automatic translated by Algorithmi 22.05
//(c) Ant0nio M@nso 	Tue, 2022 Sep 13 21:58
